pkg/arvo/lib/hood/drum.hoon

```diff
@@ -76,56 +76,42 @@
   |=  [our/ship lit/?]
   %-  ~(gas in *(set well:gall))
   ^-  (list well:gall)
+  ::  boot all default apps off the home desk
+  ::
+  =-  (turn - |=(a=term home+a))
+  ^-  (list term)
   ?:  lit
-    :~  [%home %dojo]
-        [%home %azimuth-tracker]
+    :~  %dojo
+        %azimuth-tracker
     ==
-  =+  myr=(clan:title our)
-  ::
-  ?:  ?=($pawn myr)
-  :~  [%home %lens]
-      [%base %hall]
-      [%base %talk]
-      [%base %dojo]
-      [%base %modulo]
-      [%home %launch]
-      [%home %publish]
-      [%home %clock]
-      [%home %weather]
-      [%home %group-store]
-      [%home %group-hook]
-      [%home %permission-store]
-      [%home %permission-group-hook]
-      [%home %chat-store]
-      [%home %chat-hook]
-      [%home %chat-view]
-  ==
-  :~  [%home %lens]
-      [%home %acme]
-      [%home %dns]
-      [%home %dojo]
-      [%home %hall]
-      [%home %talk]
-      [%home %modulo]
-      [%home %launch]
-      [%home %publish]
-      [%home %clock]
-      [%home %weather]
-      [%home %group-store]
-      [%home %group-hook]
-      [%home %permission-store]
-      [%home %permission-group-hook]
-      [%home %chat-store]
-      [%home %chat-hook]
-      [%home %chat-view]
-      [%home %azimuth-tracker]
+  %+  welp
```

Comment:
Does this need a welp? I feel as though I've used weld for expressions with :~ before...

Reply:
It's either `welp`, or two extra casts if you want to make `weld` work. We already have a top-level cast, so this keeps the noise down.

Reply:
gotcha